test/agentchat/test_agent_file_logging.py

```diff
@@ -0,0 +1,121 @@
+import json
+import os
+import sys
+import uuid
+
+import pytest
+
+sys.path.append(os.path.join(os.path.dirname(__file__), ".."))
+sys.path.append(os.path.join(os.path.dirname(__file__), "../.."))
+
+from conftest import skip_openai  # noqa: E402
+from test_assistant_agent import KEY_LOC, OAI_CONFIG_LIST  # noqa: E402
+
+import autogen
+import autogen.runtime_logging
+from autogen.logger.file_logger import FileLogger
+
+
+@pytest.fixture
+def logger():
+    project_dir = os.path.dirname(os.path.abspath(__file__))
+    log_dir = os.path.join(project_dir, "logs")
+    os.makedirs(log_dir, exist_ok=True)
+    log_file = os.path.join(log_dir, "test.log")
+    config = {"filename": log_file}
+    logger = FileLogger(config)
+    yield logger
+    with open(logger.log_file, "w") as f:
+        f.truncate(0)
```

Comment:
I would recommend using `tempfile` to generate temporary directories to avoid any side-effects of having random logs laying around.

Reply:
it creates a file `autogen_logs/runtime.log` within your current directory.

Reply:
@Hk669 I think Wael's suggestion is that for testing, we likely need minimize our traces and not pollute user's file system. Also, the tests might be running in parallel in the future, so you want to have isolation. So using `tempfile` to generate a temporary directory just for the duration of the test is better.

Reply:
got it. thanks